packages/cli/package.json

```diff
@@ -57,7 +57,7 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "style-loader": "^0.23.1",
-    "wc-markdown-loader": "^0.1.2",
+    "wc-markdown-loader": "^0.1.4",
```

Comment:
is there any way that **wc-markdown-loader** can follow [semver](https://semver.org/)?  I don't think it's intuitive for features to be getting added and published as patches, when typically that would happen as part of a minor.  (or major if breaking).

It's easy enough for Greenwood to pin to minors using `~` for pre 1.0.0, so as to not impact downstream users when new releases of **wc-markdown-loader** are published.